tests/test_js_signing.py

```diff
@@ -0,0 +1,55 @@
+"""
+Various tests related to validation for automated signing.
+"""
+
+from js_helper import TestCase
```

Comment:
On Tue, May 05, 2015 at 07:15:07AM -0700, Mathieu Agopian wrote:
>> @@ -0,0 +1,55 @@
>> +"""
>> +Various tests related to validation for automated signing.
>> +"""
>> +
>> +from js_helper import TestCase
>
>`from .js_helper import TestCase`

I copied this from a different testcase. Validator hasn't used 
new-style relative imports in the past, but I suppose now that 
we're on 2.7 everywhere...